pkg/arvo/sys/lull.hoon

```diff
@@ -1888,6 +1883,10 @@
       $%  [%agent [=ship name=term] =task]
           [%arvo note-arvo]
           [%pyre =tang]
+      ::
+          [%grow =case =spur =noun]
```

Comment:
I think it'll be simpler to remove `=case` here, and just let %gall manage contiguous, monotonic aeons.

Reply:
I changed this and put the question of reinflation as a question to resolve in the todo.